cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -202,9 +202,7 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate rules
 
         outputDir?.let { absoluteOutputDir ->
             // Note: This overwrites any existing EvaluatorRun from the input file.
-            val ortResultOutput = ortResultInput.copy(evaluator = evaluatorRun).apply {
-                data += ortResultInput.data
-            }.mergeLabels(labels)
+            val ortResultOutput = ortResultInput.copy(evaluator = evaluatorRun).mergeLabels(labels)
```

Comment:
Commit message. This should auto-close https://github.com/oss-review-toolkit/ort/issues/2907.

Reply:
Done.